app/controllers/api/saved_jobs_controller.rb

```diff
@@ -1,5 +1,5 @@
 class Api::SavedJobsController < ApplicationController
-  before_action :authorize!, only: [:index, :show, :create]
+  before_action :authorize!, only: [:index, :show, :create, :destroy]
```

Comment:
Are any of the endpoints in this controller unauthenticated.  If they aren't we can drop the only.